rmw_connext_xtypes_dynamic_cpp/CMakeLists.txt

```diff
@@ -24,7 +26,7 @@ ament_export_dependencies(rmw rosidl_generator_cpp)
 
 register_rmw_implementation()
 
-link_directories(${Connext_LIBRARY_DIRS})
+link_directories("${Connext_LIBRARY_DIRS}")
```

Comment:
Sounds reasonable. I guess that might apply to other locations too. Will have to try this once we have the CI job running, e.g. always build in a location with spaces (@tfoote fyi).